src/Button.css

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* Generic state for button */
+button {
+  margin: 9px;
+  background-color: #cb99c5;
+  color: white; /* Color of text displayed */
+  border-radius: 10px; /* Makes button rounded */
+  padding: 15px 25px;
+  display: inline-block;
+  vertical-align: middle;
+  border: 1px solid rgba(0, 0, 0, 0.21);
+  border-bottom-color: rgba(0, 0, 0, 0.34);
+  text-shadow: 0 1px 0 rgba(0, 0, 0, 0.15);
+  box-shadow: 0 1px 0 rgba(255, 255, 255, 0.34) inset,
+    0 2px 0 -1px rgba(0, 0, 0, 0.13), 0 3px 0 -1px rgba(0, 0, 0, 0.08),
+    0 3px 13px -1px rgba(0, 0, 0, 0.21);
+}
+
+/* Active state for button */
+button:active {
+  top: 1px;
+  border-color: rgba(0, 0, 0, 0.34) rgba(0, 0, 0, 0.21) rgba(0, 0, 0, 0.21);
+  box-shadow: 0 1px 0 rgba(255, 255, 255, 0.89), 0 1px rgba(0, 0, 0, 0.05) inset;
+  position: relative;
```

Comment:
Setting the position to absolute for the neutral button messes with it's positioning for the active and disabled states. (I tried this and the buttons are dancing on the page now!) 

![image](https://user-images.githubusercontent.com/84807533/135566036-ce57defb-fe4b-4fc9-b746-c25bd67421c4.png)
vs
![image](https://user-images.githubusercontent.com/84807533/135566218-262c2624-77af-4f97-b261-69336f382ad4.png)
Note: with even more text it looks even weirder.

If I make the position for the neutral button "relative" like the disabled and active states, this dancing button problem goes away. So should I assign the neutral button as "position: relative"?

Reply:
https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/
Since you aren't modifying the other positioning attributes, setting `position: relative` doesn't actually do anything and could be omitted. But your `button:active` and `button:disabled` moves down a single pixel (a very important move I'm sure) so they should keep their `position: relative`.